core-library/src/main/java/org/silverpeas/core/socialnetwork/relationship/RelationShipService.java

```diff
@@ -22,33 +22,35 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-package org.silverpeas.core.socialnetwork.relationShip;
+package org.silverpeas.core.socialnetwork.relationship;
 
-import java.sql.Connection;
-import java.sql.SQLException;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-
-import org.silverpeas.core.socialnetwork.model.SocialInformation;
-import org.silverpeas.core.silvertrace.SilverTrace;
+import org.silverpeas.core.exception.UtilException;
+import org.silverpeas.core.notification.system.ResourceEvent;
 import org.silverpeas.core.persistence.jdbc.DBUtil;
+import org.silverpeas.core.socialnetwork.model.SocialInformation;
 import org.silverpeas.core.util.ServiceProvider;
-import org.silverpeas.core.exception.UtilException;
+import org.silverpeas.core.util.logging.SilverLogger;
 
 import javax.inject.Inject;
 import javax.inject.Singleton;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
 
 @Singleton
 public class RelationShipService {
 
+  @Inject
+  private RelationShipDao relationShipDao;
+  @Inject
+  private RelationShipEventNotifier relationShipEventNotifier;
+
   public static RelationShipService get() {
     return ServiceProvider.getService(RelationShipService.class);
   }
 
-  @Inject
-  private RelationShipDao relationShipDao;
-
   protected RelationShipService() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1213)